--- src/crypto.ts
+++ src/crypto.ts
@@ -62,7 +62,11 @@
       return JSON.parse(decodedData);
     } catch (oldError) {
       if (encryptedBase64 !== null && encryptedBase64 !== '') {
-          return JSON.parse(encryptedBase64);
+          try {
+              return JSON.parse(encryptedBase64);
+          } catch (e) {
+              return null;
+          }
       }
       return null;
     }
